### ISSUE TITLE
Add default pausing functionality to CHS-jsPsych studies

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1299]
# TL;DR

Provide some default experiment-pausing functionality for CHS-jsPsych studies, such as the ability to pause in specific situations (key press, pause button, exiting fullscreen), and show a message/image when the study is paused and about to resume.

# Narrative

As a researcher using EFP, I'm able to use the [pause-unpause mixin](https://lookit.readthedocs.io/projects/frameplayer/en/latest/mixins/pause-unpause.html) to allow families to pause/unpause the study during specific frames, and I can customize this behavior with various parameters. This functionality is very useful when running experiments with infants.

When running jsPsych experiments on CHS, I'd like to allow families to pause my study when they need to, without any or much additional programming. Ideally the CHS-jsPsych experiment runner would have some reasonable defaults for pausing/unpausing, so that it just works 'out-of-the-box', but with customization options as well.

# Acceptance Criteria

- [ ] Tests
- [ ] Documentation
- [ ] Scoping: in the first pass for this feature, decide whether pausing parameters should be set on the study and/or trial level, and which customization options to provide for researchers (if necessary, create additional issues for things we aren't implementing now but might want to later)
- [ ] If the researcher wants to allow families to pause/unpause the study, they should be able to set a parameter for this.
- [ ] Provide reasonable defaults for pausing/unpausing (e.g. initiate with a button click and/or specific key press, display a 'study is paused' message).

# Implementation Notes

We can make use of the existing [`jsPsych.pauseExperiment()`](https://www.jspsych.org/7.3/reference/jspsych/#jspsychpauseexperiment) and [`jsPsych.resumeExperiment()`](https://www.jspsych.org/7.3/reference/jspsych/#jspsychresumeexperiment) functions.

We could consider adding an experiment-level `allow_pausing` parameter to our custom `jsPsych.init` function. However, with this approach, researchers wouldn't have the option to only allow pausing during specific trials.

Another option is to create a jsPsych pause [extension](https://www.jspsych.org/7.3/overview/extensions/) - this way, researchers could add the extension to the specific trials in which they want to allow the pausing behavior. This would also mean we could add lots of customization parameters via the extension rather than the `jsPsych.init`function, which helps to avoid cluttering up the set of init/experiment-level parameters. We could ask the jsPsych team whether they'd be interested in including a pause extension in the library, since that might appeal to many non-CHS researchers as well.
